build/Makefile.build

```diff
@@ -44,8 +45,9 @@ release:
 ifneq ($(SKIP_TAG),true)
 	@ $(MAKE) tag
 endif
-	@ echo "-> Releasing $(BINARY) v$(VERSION)..."
-	@ ./scripts/goreleaser release --rm-dist --release-notes=notes/$(VERSION).md
+	@ echo "-> Releasing $(BINARY) $(VERSION)..."
+	@ okta-awscli
```

Comment:
I don't think this is something our users will have installed :D

Reply:
We (the maintainers) are the only ones who are going to be releasing this and it's needed to obtain the necessary credentials. If we move to GitHub actions we can look into removing it.